modules/ossm-operator-install.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-operator-installation_{context}"]
+= Installing the Operators
+
+The {ProductShortName} installation process introduces an Operator to manage the installation of the control plane within the `istio-operator` namespace. This Operator defines and monitors a custom resource related to the deployment, update, and deletion of the control plane.
+
+Starting with {ProductVersion} {ProductVersion}, you must install the Jaeger Operator and the Kiali Operator before the {ProductName} Operator can install the control plane.
+
+
+[id="ossm-operator-install-jaeger_{context}"]
+== Installing the Jaeger Operator
+You must install the Jaeger Operator for the {ProductName} Operator to install the control plane.
+
+.Prerequisites
+* An account with cluster administrator access.
+
+.Procedure
+
+. Log into your {product-title} installation as a cluster administrator.
+
+. Run the following commands to install the Jaeger Operator:
++
+----
+$ oc new-project observability # create the project for the jaeger operator
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/crds/jaegertracing_v1_jaeger_crd.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/service_account.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/role.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/role_binding.yaml
+$ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/operator.yaml
+----
+
+[id="ossm-operator-install-kiali_{context}"]
+== Installing the Kiali Operator
+You must install the Kiali Operator for the {ProductName} Operator to install the control plane.
+
+.Prerequisites
+
+* Review the link:https://www.kiali.io/documentation/getting-started[Kiali] documentation.
+
+* An account with cluster administrator access.
+
+.Procedure
+
+. Log into your {product-title} installation as a cluster administrator.
+
+. Run the following command to install the Kiali Operator:
++
+----
+$ bash <(curl -L https://git.io/getLatestKialiOperator) --operator-image-version v1.0.0 --operator-watch-namespace '**' --accessible-namespaces '**' --operator-install-kiali false
```

Comment:
This only works for non-multi tenancy cases.
For MT it will be a different one

Reply:
cc @jmazzitelli

Reply:
@gbaufake If you provide me the command for MT I will update this and put a note in for MT.

Reply:
opps. I meant for my last comment to go here with this change. We want to eventually remove this `accessible namespace=**` setting.

Reply:
@jmazzitelli So for TP12, what you're saying, is I should maintain two commands -- the existing one specified for single-tenant installs, and the same command without `accessible namespace=**` for MT installations?

Reply:
@geekspertise Well, no ... it's complicated :-)

OK, so ... if Maistra is going to support both non-MT and MT mode (which it does today with its multitenancy setting in the SMCP) then the accessible_namespaces here should be **. Why? Because if the user set's Maistra "multitenancy" setting to "false", that means Kiali is going to go into "cluster" or non-MT mode too - and in that case Kiali needs these superpowers (which `accessible_namespaces=**` will end up providing). But we don't know if the user is going to use non-MT mode or MT mode at the time you install kiali operator - so we have to tell the kiali operator install script to give it those permissions "just in case" the user switches to non-MT mode.

Now, *if* Maistra eliminates support for non-MT mode, *then* we can get rid of the docs that say they should use `accessible_namespaces=**` because in that case the user will NEVER need Kiali to go into non-MT mode and so the operator will NEVER need these superpowers (so there is no need for accessible_namespaces=**.

I'm sure that is clear as mud. But in short:

1) Keep the docs as they are right now **ONLY IF** Maistra is still supporting non-MT mode.
2) Change the docs by removing this `accessible_namespaces=**` **ONLY IF** Maistra is eliminating all support for non-MT mode and will only support MT mode.

Reply:
@jmazzitelli Is anything simple? :)

1. I will leave the docs as-is for TP12 with regard to this. 
2. I will pull the accessible_namespaces string out of the docs for RC1 when MT can no longer be turned off by users.